dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala

```diff
@@ -33,53 +31,89 @@ import com.fasterxml.jackson.databind.ObjectMapper
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
 import com.fasterxml.jackson.databind.SerializationFeature
+import za.co.absa.enceladus.dao.MenasRestDAO.log
 
 object EnceladusRestDAO extends EnceladusDAO {
   val conf = ConfigFactory.load()
   val restBase = conf.getString("menas.rest.uri")
   private val userName = LoggedInUserInfo.getUserName
 
+  var sessionCookie: String = ""
```

Comment:
TBH I don't like `var`s either but I don't have a solution for it. But how about setting them `private` for the time being

Reply:
Yeah, I have an idea using closures, but they can't be private because MenasRestDao needs them too. There is no reason to keep the two separate, but I didn't want to do this heavy a refactoring with this PR.

Reply:
If you are thinking about it. Then just resolve this conversation. I believe in you.